apps/expo/src/components/top-collections/TopCollectionsHomeSection.tsx

```diff
@@ -18,7 +17,13 @@ const TopCollectionsHomeSection: FC = () => {
 
   return (
     <View>
-      <SectionHeader title="Top Collections" hasViewAll={true} />
+      <SectionHeader
+        title="Top Collections"
+        hasViewAll={true}
+        onViewAllPress={() => {
+          navigation.navigate("ViewAll", { fetchedData: "topCollections" });
+        }}
+      />
       <FlatList
```

Comment:
Top collections not centered. Try putting "self-center" on parent container

Reply:
![379403156_985348625870987_4131119955113242148_n](https://github.com/HansGabriel/TestTrek/assets/68092712/52db7e3f-e80b-4231-8d92-88bd32200b64)